File: Algorithms and Data  Structure Problems/Strings/1) amendTheSentence/amendTheSentence.cpp
std::string amendTheSentence(std::string s) {
    string replacement;
    for(int i = 0 ;  i < s.size(); ++i){
        if(s[i]>='A' && s[i]<='Z'){
            if(i!= 0){
                replacement+=" ";
            }
            replacement+=char(s[i]+32);
            continue;
        }
        replacement += s[i];
    }
    return replacement;
}

File: Algorithms and Data  Structure Problems/Strings/1) amendTheSentence/amendTheSentence.js
function amendTheSentence(s) {
    for(var i = 0; i < s.length; i++){
        if(s[i] == s[i].toUpperCase()){
            if(i != 0){
                var space = " ";
                console.log(s);
                s = [s.slice(0, i) + space + s.slice(i)].join("");
                i++
            }
        }
    }
    s = s.toLowerCase();
    return s;
}